composer.json

```diff
@@ -19,7 +19,7 @@
         "symfony/http-kernel": "~2.2",
         "symfony/dependency-injection": "~2.2",
         "symfony/templating": "~2.2",
-        "symfony/locale": "~2.2",
+        "symfony/intl": ">=2.3.34",
```

Comment:
* This is wrong. With that, any version of Symfony could be included, even `3.0`. Use `^` notation instead
* Please do not simply require the last `2.3.x` version. You have to found the lowest compatible version.